SpotifyAPI/Web/Auth/WebApiFactory.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using SpotifyAPI.Web.Enums;
+using SpotifyAPI.Web.Models;
+
+namespace SpotifyAPI.Web.Auth
+{
+    public class WebApiFactory
+    {
+        private readonly string m_RedirectUrl;
+        private readonly int m_ListeningPort;
+        private readonly string m_ClientId;
+        private readonly TimeSpan m_Timeout;
+        private Scope m_Scope;
```

Comment:
Could you adapt this names to the code-style?
`private readonly string _redirectUrl`